go.mod

```
@@ -1,6 +1,6 @@
 module github.com/k3s-io/k3s
 
-go 1.20
+go 1.21
```

Upstream is still on `go 1.20`, can we leave this as-is?
https://github.com/kubernetes/kubernetes/blob/release-1.27/go.mod#L9

```suggestion
go 1.20
```